p2p/message.go

```diff
@@ -132,6 +135,7 @@ func (m *transportMsg) writeTo(conn *textproto.Writer) error {
 	header.WriteString(fmt.Sprintf("%s:%d\r\n", headerFieldRequestID, m.id))
 	header.WriteString(fmt.Sprintf("%s:%s\r\n", headerFieldTopic, m.topic))
 	header.WriteString(fmt.Sprintf("%s:%d\r\n", headerStatusCode, m.statusCode))
+	header.WriteString(fmt.Sprintf("%s:%s\r\n", headerMsg, m.msg))
```

Comment:
It might be considered a vulnerability of exposing internal error details to a peer. But good for debugging :) Maybe we should make it configurable and enabled by default? :)

Reply:
Let's make it in as a separate issue if you really think that it could be a vulnerability.